rules.ini

```diff
@@ -101,7 +101,10 @@ GDevelop[] = (?:^|/)LICENSE\.GDevelop\.txt$
 GDevelop[] = (?:^|/)gdjs-evtsext__
 GDevelop[] = (?:^|/)GDCpp\.dll$
 Gideros = gideros
-Glacier = \.rpkg$
+Glacier[] = \.rpkg$
+Glacier[] = \.pc_packagelist$
+Glacier[] = ^(?:Scenes)\/(?:.+\/)?[^\/]+\.(?:WHD)$
+Glacier[] = ^Music\/Low\/[^\/]+\/[^\/]+\.sty$
```

Comment:
these 2 regex are unreadable, this won't be merged in this current state
And `.+` / any usage of + is something that must be avoided

Reply:
Will fix when I get the chance, sorry 

I know it's a mess lol

Reply:
Fixed

Reply:
Okay this is extremely annoying, SteamDB doesnt support wildcards?

Reply:
using .* to replace .+ is not a fix, it's the same things but also allowing 0 caracters